### ISSUE TITLE
Web Interface not reachable

### ISSUE THREAD

**Issue by JonasKruse1** [jomjol/AI-on-the-edge-device#3076]
### The Problem

Hi, I configured the ESP Cam a couple of days ago and everthing worked out fine but today the web interface is not reachable but if I ping it via CMD it works just fine 

### Version

latest 

### Logfile

```shell
I can't access the log files
```

### Screenshots

![image](https://github.com/jomjol/AI-on-the-edge-device/assets/92718910/fcc1052c-cac2-44cf-a076-c46733bf2041)
![image](https://github.com/jomjol/AI-on-the-edge-device/assets/92718910/259148ea-8de0-49df-9715-48650cf9668e)

**Comment by JonasKruse1** [comment]
Hi, I attached the screenshot from my fritzbox its -65dBm 
![image](https://github.com/jomjol/AI-on-the-edge-device/assets/92718910/bf4f3080-07fc-461f-8ff5-5105012bcced)
and I also tried all the links above none worked

**Comment by SybexX** [comment]
have you tried with another pc or did you briefly disconnect the ESP from the power?
Some people have reported that their virus software blocked the site, others that they had problems with VPN.

**Comment by freese0711** [comment]
The same with my module.

After a ping the webinterface of the device can be reached for about 10 minutes. Configuration can be done.
After that time is can not be reached anymore but the module works fine. I get values via MQTT.

**Comment by JonasKruse1** [comment]
> have you tried with another pc or did you briefly disconnect the ESP from the power? Some people have reported that their antivirus software blocked the site, others that they had problems with VPN.

Yes I have tried it all but I can't access the website, not with chrome not with firefox not with edge and tried restarting multiple times

**Comment by SybexX** [comment]
Then I would say there is something wrong with your SD card.
Either something has destroyed some files or the SD card is broken(shortly from total failure).
